config/crd/bases/delivery.ocm.software_componentversions.yaml

```diff
@@ -48,40 +48,39 @@ spec:
           metadata:
             type: object
           spec:
-            description: ComponentVersionSpec defines the desired state of ComponentVersion
+            description: ComponentVersionSpec specifies the configuration required
+              to retrieve a component descriptor for a component version.
             properties:
               component:
-                description: 'Every Component Version has a name. Name and version
-                  are the identifier for a Component Version and therefor for the
-                  artifact set described by it. A component name SHOULD reference
-                  a location where the component’s resources (typically source code,
-                  and/or documentation) are hosted. It MUST be a DNS compliant name
-                  with lowercase characters and MUST contain a name after the domain.
-                  Examples: - github.com/pathToYourRepo'
+                description: Component specifies the name of the ComponentVersion.
                 type: string
               interval:
+                description: Interval specifies the interval at which the Repository
+                  will be checked for updates.
                 type: string
               references:
+                description: References specifies configuration for the handling of
+                  nested component references.
                 properties:
                   expand:
                     type: boolean
```

Comment:
1. explanation/description for expand is missing and indicate by default it is false
2. Also add explanation about direct relation to nested components and resourceRef. If detail is not added here, it should be there somewhere to explain the linkage